contracts/GenArt721CoreV3.sol

```diff
@@ -18,6 +18,17 @@ pragma solidity 0.8.9;
 contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
     event ProjectCompleted(uint256 indexed _projectId);
 
+    event ProposedArtistAddressesAndSplits(
+        uint256 indexed _projectId,
+        address _artistAddress,
+        address _additionalPayeePrimarySales,
+        uint256 _additionalPayeePrimarySalesPercentage,
+        address _additionalPayeeSecondarySales,
+        uint256 _additionalPayeeSecondarySalesPercentage
+    );
+
+    event AcceptedArtistAddressesAndSplits(uint256 indexed _projectId);
```

Comment:
~Should we also emit the hash of the accepted proposal here, to allow for reconciliation in the case of multiple proposals where only the most recent is accepted?~

I think my initial comment isn't necessary because we have the mapping based on projectId so there can only be one proposal at a time, but wanted to flag anyway for discussion just in case I'm overlooking antyhing.

Reply:
Yes you are correct - artist can only have a single set of proposed values at a time. Additionally, since we store and check the hash of the set of values when admin approves, artists cannot do anything weird like front-run our approval transaction and cause us to accidentally approve a different set of values than we intended to approve.

Reply:
Yeah that makes sense–I dig this design. cc @yoshiwarab on this comment thread as the second reviewer.